modules/core/src/test/scala/org/tessellation/infrastructure/snapshot/GlobalSnapshotConsensusFunctionsSuite.scala

```diff
@@ -92,19 +110,24 @@ object GlobalSnapshotConsensusFunctionsSuite extends MutableIOSuite with Checker
       ???
 
   }
-  val env: AppEnvironment = AppEnvironment.Mainnet
+
+  val env: AppEnvironment = AppEnvironment.Testnet
```

Comment:
we should revert this to Mainnet as soon as the SC would be available on mainnet. could you create a ticket in jira, not to forget about that? imo we should test against the production environment whenever it's possible

Reply:
Ticket created.